src/routes/api/chat/+server.ts

```diff
@@ -0,0 +1,20 @@
+import { createOpenAI } from '@ai-sdk/openai';
+import { convertToCoreMessages, streamText } from 'ai';
+import type { RequestHandler } from './$types';
+import { env } from '$env/dynamic/private';
+
+const groq = createOpenAI({
+  baseURL: 'https://api.groq.com/openai/v1',
+  apiKey: env.GROQ_API_KEY ?? '',
+});
+
+export const POST = (async ({ request }) => {
+  const { messages } = await request.json();
+
+  const result = await streamText({
+    model: groq('llama-3.1-8b-instant'),
+    messages: convertToCoreMessages(messages),
```

Comment:
please add system prompt and explain it in the article

for example something like:
  system:
    `You are helpful assistant from Daytona with senior level expertise in development. ` +
    `Respond to the users' request with a clear and concise answer ` +
    `about any of his development or devops problems.`,

Reply:
sure, i will add system prompt and make sure of these changes.